lib/whoop/formatters/sql_formatter.rb

```diff
@@ -39,8 +50,20 @@ def self.format(sql, colorize: false, explain: false)
       def self.generate_pretty_sql(sql)
         rule = AnbtSql::Rule.new
         rule.indent_string = "  "
+
         formatter = AnbtSql::Formatter.new(rule)
-        formatter.format(sql.dup)
+        formatted_string = formatter.format(sql.dup)
+
+        # Anbt injects additional spaces into joined symbols.
+        # This removes them by generating the "broken" collection
+        # of symbols, and replacing them with the original.
+        PATTERNS_TO_PRESERVE.each do |token, pattern|
+          next unless formatted_string.include?(pattern)
+
+          formatted_string.gsub!(pattern, token.to_s)
+        end
```

Comment:
The variable name `token` doesn't seem to make sense in this context. As I understand it, the first argument (key) is the correct pattern, and the 2nd argument (value) is the incorrect pattern.

I would consider renaming both to `correct` and `incorrect`, or something like that.